ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -198,7 +198,7 @@ spec:
 EOF
     set +e 
     COUNTER=0
-    while [ $COUNTER -lt 30 ]
```

Comment:
You want the `COUNTER` stand for the total timeout or the retry times?

Reply:
Yes,